cypress/integration/app_admin/app_admin.js

```diff
@@ -0,0 +1,49 @@
+import { Given, When, Then, Before, After } from 'cypress-cucumber-preprocessor/steps';
+
+const USER_NAME = 'test@unit.no';
+const APP_ADMIN_ROLE = 'App-admin';
+
+Before(() => {
+  cy.deleteUser(USER_NAME).then(() => {
+    cy.addUser(USER_NAME, APP_ADMIN_ROLE).then((userId) => {
+      cy.wrap(userId).as('userId');
+      cy.visit('/');
+      cy.wait(5000);
+
+      cy.get('[data-testid=create-author-button]').click();
+      cy.get('[data-testid=modal_next]').click();
+      cy.get('[data-testid=skip-connect-to-orcid]').click();
+      cy.get('[data-testid=menu]').click();
+
+      // Set language
+      cy.get('[data-testid=menu-user-profile-button]').click();
+      cy.get('[data-testid=language-selector]').click();
+      cy.get('[data-testid=user-language-eng]').click();
+      cy.visit('/');

```

Comment:
Ser for meg at dette må gjøres på flere tester, så kanskje lurt å lage en command av det?

Reply:
Jepp, er den tasken jeg jobber med nå.

Reply:
Skal det inn i denne PRen eller tar du det i en ny en?

Reply:
Fikser det nå.